pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/rule/CyclomaticComplexityRule.java

```diff
@@ -28,21 +28,21 @@
  */
 public class CyclomaticComplexityRule extends AbstractJavaMetricsRule {
 
-    public static final IntegerProperty REPORT_LEVEL_DESCRIPTOR = new IntegerProperty(
+    private static final IntegerProperty REPORT_LEVEL_DESCRIPTOR = new IntegerProperty(
```

Comment:
This would imply subclasses are not allowed to interrogate these properties using the descriptors.  Is this intended? Is subclassing not permitted (the class is not `final`)?